src/validation.cpp

```diff
@@ -4154,13 +4154,14 @@ VerifyDBResult CVerifyDB::VerifyDB(
     }
     if (skipped_l3_checks) {
         LogPrintf("Skipped verification of level >=3 (insufficient database cache size). Consider increasing -dbcache.\n");
+        return VerifyDBResult::ERROR_INSUFFICIENT_CACHE;
```

Comment:
Done - decided to use an if instead of ternary because `skipped_no_block_data` is treated similarly.
I decided to give `skipped_l3_checks` precedence - so if in addition to this, `skipped_no_block_data` is also true, we'd  report `VerifyDBResult::SKIPPED_L3_CHECKS`.